server/.env

```diff
@@ -0,0 +1 @@
+ACCESS_TOKEN_SECRET=ee272787deefed0bbce296b8a325bdca3c76ecbf1e2f980748035ee5e048811dd7baa719fa97a8f3a634092f28fd9d90506c0d0669d667b54bc9b12cfa56134e
```

Comment:
xD
please add **.env** to your gitignore file
you can make an example of this file (**.env-example**), to show what values are needed

Reply:
I forgot about gitignore ups xd
I deleted .env, pushed, created it once again and added to .gitignore
also created .env-example